themes/scss/map/dashboard-info.scss

```diff
@@ -235,18 +232,18 @@
 }
 @include media-query-mobile() {
   .CDB-Dashboard-infoInner {
-    transform: translateX(-100%);
-    overflow: visible;
-    z-index: 2;
     right: 40px;
     width: auto;
     max-width: 100%;
+    transform: translateX(-100%);
+    overflow: visible;
+    z-index: 2;
   }
   .CDB-Dashboard-info.is-active {
     .CDB-Dashboard-infoInner {
       width: auto;
-      box-shadow: 300px 0 0 rgba(0, 0, 0, 0.45);
       transform: translateX(0);
+      box-shadow: 300px 0 0 rgba(0, 0, 0, 0.45);
```

Comment:
Color literals like `rgba(0, 0, 0, 0.45)` should only be used in variable declarations; they should be referred to via variable everywhere else.